### PR TITLE
fix issues #2 and issues #3

### PR DESCRIPTION
**Pull request by liyujiang-gzu**
fix issues #2 and issues #3

### CODE REVIEW

File: lib/src/licenses.dart [hunk 1]
```diff
@@ -133,15 +133,24 @@ class _LicenseListPageState extends State<LicenseListPage> {
       String exerpt;
       for (LicenseEntry license in lisenses) {
         if (license.packages.contains(package)) {
-          exerpt = license.paragraphs.first.text.split('.').first;
+          final String p = license.paragraphs.first.text.trim();
+          // Third paty such as `asn1lib`, the license is a link
```
Comment (DavBfr) [inline lib/src/licenses.dart]: Third party: missing a 'r'

**Comment by DavBfr** [comment]
Is it really useful to have the full Android example?

**Comment by liyujiang-gzu** [comment]
> Is it really useful to have the full Android example?

Please show me how to run the screenshot without a full Android example

**Comment by DavBfr** [comment]
You can just run `flutter create example` to generate the missing files

**Comment by liyujiang-gzu** [comment]
> You can just run `flutter create example` to generate the missing files

Thanks!

**Pull request merged**